baseplate/lib/metrics.py

```diff
@@ -135,16 +145,16 @@ def __init__(self, transport: Transport, namespace: str):
         self.transport = transport
         self.namespace = namespace.encode("ascii")
 
-    def timer(self, name: str) -> "Timer":
+    def timer(self, name: str, tags: Optional[Dict] = None) -> "Timer":
```

Comment:
Should the tag dict be `Dict[str, Any]`?  If so' it might be good to use that and define it as `TagDict = Dict[str, Any]`.

Alternatively, would keyword args make sense?

Reply:
I think Dict[str, Any] is fine, but it does have to stay Optional

Reply:
If you want to default to an empty value, yes, because you want to use `None`.  You don't want to use an empty dict as a default value (or any mutable container).  The reason being that "default" value is a global value so if you mutate it, that mutation applies to the "default" so anytime you use it after that, it has those values set.

Reply:
Actually, I think it should be Dict[str,str], unless you think we should be able to accept booleans as values?